Directory.Packages.props

```diff
@@ -11,19 +11,20 @@
     <PackageVersion Include="Microsoft.Build.Tasks.Core" Version="$(MicrosoftBuildPackageVersion)" />
     <PackageVersion Update="Microsoft.Build.Tasks.Core" Version="16.9.0" Condition="'$(TargetFramework)' == 'netstandard2.0'" />
     <PackageVersion Update="Microsoft.Build.Tasks.Core" Version="15.9.20" Condition="'$(TargetFramework)' == 'net46'" />
+	<PackageVersion Include="Microsoft.TestPlatform" Version="$(MicrosoftBuildPackageVersion)" />
     <PackageVersion Include="Microsoft.Win32.Registry" Version="5.0.0" />
     <PackageVersion Include="Newtonsoft.Json" Version="13.0.3" />
     <PackageVersion Include="System.CodeDom" Version="7.0.0" />
     <PackageVersion Include="System.Threading.Tasks.Dataflow" Version="6.0.0" Condition=" '$(TargetFramework)' != 'net46' " />
     <PackageVersion Include="System.Threading.Tasks.Dataflow" Version="4.11.1" Condition=" '$(TargetFramework)' == 'net46' " />
   </ItemGroup>
   <ItemGroup Condition="'$(IsTestProject)' == 'true'">
-    <PackageVersion Include="AssemblyShader" Version="1.0.3-preview" />
-    <PackageVersion Include="Microsoft.NET.Test.Sdk" Version="17.7.2" />
-    <PackageVersion Include="MSBuild.ProjectCreation" Version="10.0.0" />
-    <PackageVersion Include="Shouldly" Version="4.2.1" />
-    <PackageVersion Include="xunit" Version="2.6.0" />
-    <PackageVersion Include="xunit.runner.visualstudio" Version="2.4.5" />
+	  <PackageVersion Include="AssemblyShader" Version="1.0.3-preview" />
+	  <PackageVersion Include="Microsoft.NET.Test.Sdk" Version="17.7.2" />
+	  <PackageVersion Include="MSBuild.ProjectCreation" Version="10.0.0" />
+	  <PackageVersion Include="Shouldly" Version="4.2.1" />
+	  <PackageVersion Include="xunit" Version="2.6.0" />
+	  <PackageVersion Include="xunit.runner.visualstudio" Version="2.4.5" />  
```

Comment:
These packages are pretty specific to some of your usages. Why not `FluentAssertions`? Why not MSTest or NUnit?

To me you should have something closer to what arcade is doing and condition some of the package deps on the framework.

Shoudly, FluentAssertions... are specific assertion libraries that should also be conditioned by something else. For example, I can use xunit and xunit assert, or xunit and shoudly...

Reply:
These are not included in the package.